Monika After Story/game/script-topics.rpy

```diff
@@ -3147,6 +3147,25 @@ label monika_ribbon:
     m 1eua "You should do whatever makes you feel better about yourself."
     m 1hua "I'm the only other person you need, anyways, and I'll love you no matter what you look like."
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_outdoors",category=['outdoors'],prompt="Camping safety",random=True))
+
+label monika_outdoors:
+    m 1eua "Do you ever go camping, [player]?"
+    m 3eua "It's a wonderful way to relax and get fresh air."
+    m 3eud "But, there are several dangers that most don't think about."
+    m 3lud "Such as bug spray, or sunscreen. Many people forget, or even forgo them, thinking them unimportant."
+    m 1eud "But without it, sunburns are almost inevitable, and many insects carry diseases that can truly harm you."
+    m "So, please promise me that the next time you go outdoors, you won't forget them."
+    m "It may be inconvenient, but if you don't use them, you many up miserable, or even get really sick."
+    m 1eub "On the bright side..."
+    m "Once I cross over, if you remember to bring the sunscreen..."
+    m 1tfu "I might need help putting it on."
+    m 1hub "Ahahahaha!"
+    m 1efu "I'm only teasing you, [player]."
+    m 1tfu "Well, at least a little."
```

Comment:
I'd add an "ehehe" to end with.

Reply:
Okie dokie. Those are all easy tweaks, and I'll see if I can expand it a bit, as well as reference the backpacking option.

And, I got several outdoors-esque ideas, but I can change the category.

Reply:
Drat. Looked at the backpacking one; It's great, but it possibly cuts out the lake idea I had. XP

Reply:
Wait, I might be able to make it work.